### DISCUSSION TITLE
Improved map preview

### DISCUSSION THREAD

**Discussion by penfriendz** [TheSuperHackers/GeneralsGamePatch#2555] [Ideas]
better idea of what tech buildings are on minimap
instead of star, have oil derrick, oil refinery, reinforcement pad, etc show up

**Comment by ExiLe-ZH** [comment]
Maybe unrelated, but in loading screen preview. Instead of a player number, it would be nice to have a team number and army icon.

**Comment by xezon** [comment]
This should be an issue report, with enhancement label - if it does not yet exist.
